feconf.py

```diff
@@ -213,6 +213,15 @@
 # for recently published explorations in /library/recently_published page.
 RECENTLY_PUBLISHED_QUERY_LIMIT_FULL_PAGE = 20
 
+# The current version of the Android feedback report info blob schema.
+CURRENT_ANDROID_REPORT_SCHEMA_VERSION = 1
+
+# The current version of the web feedback report info blob schema.
+CURRENT_WEB_REPORT_SCHEMA_VERSION = 1
+
+# The current version of the app feedback report daily stats blob schema.
+CURRENT_REPORT_STATS_SCHEMA_VERSION = 1
```

Comment:
REPORT is quite generic, might need the "APP_FEEDBACK" prefix.

Ditto for REPORT_SCRUBBER_BOT_ID and its corresponding user ID.

Reply:
Ack, I was trying to find something that wouldn't be too wordy. Is `CURRENT_WEB_REPORT_SCHEMA_VERSION` and `CURRENT_ANDROID_REPORT_SCHEMA_VERSION` ok to keep? `CURRENT_WEB_APP_FEEDBACK_REPORT_SCHEMA_VERSION` seems to wordy.

Reply:
Yes, the others are fine IMO.